apps/heureka/src/hooks/useUrlState.js

```diff
@@ -74,7 +80,17 @@ const useUrlState = () => {
     if (JSON.stringify(updatedState) !== JSON.stringify(currentState)) {
       urlStateManager.push(updatedState)
     }
-  }, [loggedIn, detailsFor, activeView])
+  }, [
+    loggedIn,
```

Comment:
Are you still using loggedIn? I thought all authentication-related code was removed.

Reply:
No, loggedIn should be deleted. Thanks for the reminder.